recipe/meta.yaml

```diff
@@ -1,23 +1,24 @@
-{% set version = "0.12.1" %}
+{% set version = "0.13.1-20180305" %}
+{% set sha256 = "2a136451a7932d80b7d197b10441e26e39428d67b1443ec43bbba824705e1123" %}
```

Comment:
No need for this Jinja variable, just add the Sha below.

Reply:
will regro-cf-autotick-bot pick it up if its in the body of the meta.yaml?

Reply:
Yep. And it makes the recipe clear and readable.

Reply:
okay I will revert :)